Libplanet/Net/NetMQTransport.cs

```diff
@@ -378,9 +382,24 @@ public async Task RunAsync(CancellationToken cancellationToken)
                 await Task.WhenAll(tasks);
                 _logger.Verbose("Update complete.");
             }
-            catch (DifferentAppProtocolVersionException)
+            catch (DifferentAppProtocolVersionException e)
             {
-                _logger.Debug($"Different version encountered during {nameof(AddPeersAsync)}().");
+                AppProtocolVersion expected = e.ExpectedVersion, actual = e.ActualVersion;
+                _logger.Debug(
+                    $"Different version encountered during {nameof(AddPeersAsync)}().\n" +
+                    "Expected version: {ExpectedVersion} ({ExpectedVersionExtra}) " +
+                    "[{ExpectedSignature}; {ExpectedSigner}]\n" +
```

Comment:
I don't know how much it will be used, but how about adding a `ToString()` method to `AppProtocolVersion`?

Reply:
`AppProtocolVersion` already has `ToString()` method, but for the sake of convenience, I made it to omit signature and signer that lengths. Wouldn't it be inconvenient even if every time its string representation is longer than ~250 characters?

Reply:
Okay, make sense.